<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Чаты</string>
  </property>

  <widget class="QWidget" name="centralwidget">

   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>250</width>
      <height>600</height>
     </rect>
    </property>
   </widget>

   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>620</width>
      <height>450</height>
     </rect>
    </property>
   </widget>

   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>500</width>
      <height>40</height>
     </rect>
    </property>
   </widget>

   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>470</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>

  </widget>

  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
